### DISCUSSION TITLE
Special question: Link dependencies with other reference (e.g. readme.md)

### DISCUSSION THREAD

**Discussion by Scrat94** [renovatebot/renovate#25322] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.20.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Dear Renovate Team,
first of all, thanks for this awesome project. Renovate is a game changer in our daily operations! 

Now, I have a strange and maybe extraordinary question. I have a private python project with a requirements.txt where my dependencies are successfully updated automatically. Additionally I have a `readme.md` file (badge) and a `Dockerfile` where also the same package is referenced, which is already in the requirements.txt. I want to keep all three version references of the same package in sync.

While I can use the customManager with regex to automatically update the Dockerfile and readme values as well, it creates a unncessary noice in the PR, since it will list the same dependency three times (requirements.txt, dockerfile, readme.md). 

Is it, somehow, possible to only display the dependency once (requirements.txt) in PR and link it with other changes in other files in my git repository? So if I merge the PR, the changes are applied in all three files, but on the PR itself it is only displayed once. 

### What you tried so far.

I tried to play around with customManager and regex and the update works perfectly fine. It just creates too much noise in the PR itself. 

I hope you understand my requirements. Please let me know if this makes sense or if this is maybe even already available.

**Comment by rarkins** [comment, marked as answer]
You could probably achieve this with postUpgradeTasks, where you can execute custom commands of your choosing, such as syncing the requirements.txt value to the other files
